cjs/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare const globaThis: any;
```

Comment:
this file won't actually be used, but I put it here to be consistent between the two